gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EntityTagsController.java

```diff
@@ -52,4 +55,50 @@ public Map get(HttpServletRequest request) {
 
     return entityTagsService.get(id);
   }
+
+  @ApiOperation(value = "Deletes a subset of tags for the provided entityId",
+                notes = "multiple tags can be deleted for an entity using a comma as a separator, e.g. /tag1,tag2")
+  @RequestMapping(value = "/tags/{entityId}/{tag}", method = RequestMethod.DELETE)
+  public Map delete(@PathVariable String entityId, @PathVariable String tag) {
+    Map<String, Object> operation = new HashMap<>();
+    List<Map<String, Object>> jobs = new ArrayList<>();
+    Map<String, Object> job = new HashMap<>();
+    job.put("type", "deleteEntityTags");
+    job.put("entityId", entityId);
```

Comment:
Formally, this isn't the `entityId` ... specifically it's the `entityTagsId`.

`deck-main-v001` vs `aws:deck-main-v001:prod:us-west-1`